.github/workflows/dependabot-pr.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           echo "crate_name: $crate_name, old_version: $old_version, new_version: $new_version"
 
           ./scripts/cargo-for-all-lock-files.sh -- update -p $crate_name:$old_version --precise $new_version || \
-          ./scripts/cargo-for-all-lock-files.sh -- update -p $crate_name --precise $new_version
+          ./scripts/cargo-for-all-lock-files.sh -- tree
```

Comment:
well, `tree` won't update these crates: https://github.com/anza-xyz/agave/blob/08c658dc5548e9c7a782db5f93dfd9acaa3969ed/programs/sbf/Cargo.toml#L11-L55

as far as i looked the screenshot, I think skipping `cargo update` on `/Cargo.{toml/lock}` (the root one) might be better, just like the previous impl did like this: https://github.com/anza-xyz/agave/pull/1466/files#r1616474271

After all, the root one should already handled by dependabot correctly...

Reply:
what do you think if we only use tree command here? actually, neither `update` nor `tree` updates Cargo.toml. we need other tools to do it. I guess we only care about lock file here. wdyt if we only keep the `tree` command?

Reply:
nvm. I think you're taking about if dependabot didn't update those crates correctly.

I think maybe just put `tree` command as the last command. [cb547d9](https://github.com/anza-xyz/agave/pull/1498/commits/cb547d9c9bffdd896c0f3fc40f08a687e3f6d894)

Reply:
yeah, `update` => `update` => `tree` would work too.